deny.toml

```diff
@@ -281,6 +281,10 @@ include-dependencies = true
 name = "redis-module-macros"
 allow-globs = ["*"]
 
+[[bans.build.bypass]]
+name = "redis-module-macros-internals"
+allow-globs = ["*"]
```

Comment:
How come it was not needed before?

Reply:
I don't know. But there are only build scripts so far. I think we should patch cargo-deny to be able to exclude build scripts from the binary detection. This has also been the case a few weeks ago with v8-rs where I suggested not creating the .so files too.

Reply:
But I did not see there was any build failures. So I did you sudenly concluded it is needed?

Reply:
Because I see those locally.